packages/docs/src/routes/docs/(qwikcity)/pages/index.mdx

```diff
@@ -142,3 +142,53 @@ export const head: DocumentHead = ({ head }) => {
   };
 };
 ```
+
```

Comment:
Thanks @nhayhoc , can you write a small description of this case pls? Why is needed etc. etc.

Reply:
Sometimes, I need to embed some custom JavaScript codes in the header without wanting to lazy load them via Partytown (for instance, setting up some global JavaScript variables for third-party analytics), and I need to place the Google Data Struct (this data structure assists in generating more search results for Google) into the head tag (although placing it in the body tag is also feasible). However, most other documents and websites place it in the head tag, and I believe anyone concerned with SEO would need to include ld-json documents.

Reply:
This makes sense to me. Let's write this explanation in the docs